docs/agave/guides/files/introduction.rst

```diff
@@ -553,7 +537,7 @@ Copying files and directories
 
 .. code-block:: plaintext
 
-   files-copy -D $DESTPATH -S api.tacc.cloud $PATH
+   tapis files copy AGAVE_URI, DESTINATION
```

Comment:
Is AGAVE_URI, DESTINATION correct? Below it is listed without the comma

Reply:
removed the comma